src/transformers/models/bert/modeling_bert.py

```diff
@@ -417,36 +440,52 @@ def forward(
         head_mask=None,
         encoder_hidden_states=None,
         encoder_attention_mask=None,
+        past_key_value=None,
         output_attentions=False,
     ):
+        self_attn_past_key_value = past_key_value[:2] if past_key_value is not None else None
         self_attention_outputs = self.attention(
             hidden_states,
```

Comment:
Also, while we're at it -> maybe switch to only keyword args here? It's usually safer this way